afisha-app/src/app/app.routes.ts

```diff
@@ -3,10 +3,12 @@ import {Error404Component} from "./error/error404.component";
 import {EventsListComponent} from "./events/events-list.component";
 import {EventComponent} from "./events/event.component";
 import {EventRouteActivatorService} from "./events/services/event-route-activator.service";
+import {PATH} from "./shared/constants/path.constant";
 
 export const appRoutes: Routes = [
-  { path: 'events', component: EventsListComponent },
-  { path: 'events/:id', component: EventComponent, canActivate: [EventRouteActivatorService] },
-  { path: '404', component: Error404Component },
-  { path: '', redirectTo:'/events', pathMatch: 'full'}
+  { path: PATH.EVENTS, component: EventsListComponent },
+  { path: PATH.EVENTS_ID, component: EventComponent, canActivate: [EventRouteActivatorService] },

```

Comment:
EventRouteActivatorService is a Guard. Please rename and move to appropriate folder.